userspace/libsinsp/filter.cpp

```diff
@@ -2200,7 +2229,19 @@ bool sinsp_evttype_filter::run(sinsp_evt *evt, uint16_t ruleset)
 		}
 	}
 
-        list<filter_wrapper *> *filters = m_filter_by_evttype[evt->m_pevt->type];
+
+	uint16_t etype = evt->m_pevt->type;
+
+	list<filter_wrapper *> *filters = m_filter_by_evttype[etype];
```

Comment:
Can you move this line in the else branch of the if?

Reply:
did you mean declare it on its current line but only set it to `m_filter_by_evttype[etype]` by adding an else clause on line ~2244 ?

Reply:
Yes, just to make the code a bit more clear.

Reply:
Ok, will do.